scm/src/test/java/com/sandboni/core/scm/resolvers/cli/GitDiffRunnerTest.java

```diff
@@ -37,17 +35,13 @@
     public void setup() {
         repository = GitRepository.buildRepository(GitHelper.openCurrentFolder());
         revisionResolver = new RevisionResolver(repository);
+        PowerMockito.spy(GitDiffRunner.class);
     }
 
     @Test
-    public void testDiff() throws IOException, SourceControlException {
-        mockStatic(ProcessRunner.class);
-
-        List<String> diffLines = ResourceFileUtils.getResourceFileContentAsList(getClass(), "gitDiff.out");
-
-        when(ProcessRunner.runCommand(any(), anyVararg())).thenReturn(diffLines);
-
-        RevisionScope<ObjectId> scope = revisionResolver.resolve("fc776fe5e50", "e6f7c3d2954d6"); // any
+    public void testDiff() throws Exception {
+        RevisionScope<ObjectId> scope = revisionResolver.resolve("ec302a13", "60075ec4"); // any
+        PowerMockito.doNothing().when(GitDiffRunner.class, "buildDiffCommandHelper", Mockito.any());
```

Comment:
can we add a comment before this line explaining why we need this?
// adding -- "**/*.java" "**/*.feature" fails on travis build. we don't know why